frontend_vue/src/components/tokens/types.ts

```diff
@@ -208,7 +208,7 @@ export type HitsType = {
   merchant?: string | null;
   mail?: string | null;
   referer?: string | null;
-  location?: string | GeolocationPosition | null;
+  location: string | GeolocationPosition | null;
```

Comment:
Should this not stay `location?`? 
Looking at a previous commit it has historically been marked `?` on the `HitsType` but not `BasicInfoType`:
https://github.com/thinkst/canarytokens/blob/baab54f760b08fad4da2b46a10b3bf0e4659c3fe/frontend_vue/src/components/tokens/types.ts#L177-L207